src/buildblock/ProjDataInMemory.cxx

```diff
@@ -88,34 +88,54 @@ ProjDataInMemory(shared_ptr<ExamInfo> const& exam_info_sptr,
   }
 }
 
-ProjDataInMemory::
-ProjDataInMemory(const ProjData& proj_data)
-  : ProjDataFromStream(proj_data.get_exam_info_sptr(),
-		       proj_data.get_proj_data_info_ptr()->create_shared_clone(), shared_ptr<iostream>())
+static
+shared_ptr<std::iostream>
+create_stream()
 {
+  shared_ptr<std::iostream> output_stream_sptr;
 #ifdef STIR_USE_OLD_STRSTREAM
   const size_t buffer_size = get_size_of_buffer();
```

Comment:
actually a call to a member? I guess can't have it as `static`

Reply:
Ok, no longer static. If you're happy and Travis is happy, I guess this can be merged.